modules/ingest-common/src/test/java/org/opensearch/ingest/common/RemoveProcessorTests.java

```diff
@@ -180,5 +180,21 @@ public void testRemoveMetadataField() throws Exception {
                         assertThat(ingestDocument.hasField(metadataFieldName), equalTo(false));
                     }
         }
+
+        // test remove _id when _version_type is null
```

Comment:
Could you please extract it into separate test method (if possible)?

Reply:
I've added a new test method for it.